compose.yml

```diff
@@ -0,0 +1,79 @@
+version: '3' 
+
+services:
+
+  bitcoind:
+    image: kylemanna/bitcoind:latest
+    volumes:
+      - data:/bitcoin/.bitcoin
+    ports: 
+      - "8332:8332"                                        # mainnet
+      - "18332:18332"                                      # testnet
+    command: 
+      - "-chain=${COUNTERPARTY_NETWORK:-main}"             # export COUNTERPARTY_NETWORK=test for testnet
+      - "-rpcallowip=0.0.0.0/0"
+      - "-rpcbind=0.0.0.0"
+      - "-rpcuser=rpc"
+      - "-rpcpassword=rpc"
+      - "-listen=1"
+      - "-server=1"
+      - "-printtoconsole=1"
+      - "-addresstype=legacy"
+      - "-txindex=1"
+      - "-prune=0"
+      - "-dbcache=4000"
+      - "-mempoolfullrbf=1"
+
+
+  addrindexrs:
+    image: counterparty/addrindexrs:v0.4.6
+    links:
+      - bitcoind
+    volumes:
+      - data:/root/.bitcoin
+      - data:/data
+    ports:
+      - "8432:8432"                                        # mainnet
+      - "18432:18432"                                      # testnet
+    command:
+      - "--network=${COUNTERPARTY_NETWORK:-main}"          # export COUNTERPARTY_NETWORK=test for testnet
+      - "--indexer-rpc-host=0.0.0.0"
+      - "--daemon-rpc-host=bitcoind"
+      - "--cookie=rpc:rpc"
+      - "-vvv"
+      - "--db-dir=/data/"
+    environment:
+      - ADDRINDEXRS_JSONRPC_IMPORT=${ADDRINDEXRS_JSONRPC_IMPORT:-false}
+
+
+  counterparty-core:
+    image: counterparty/counterparty:v10.0.0-rc.1
+    links:
+      - bitcoind
+      - addrindexrs
+    volumes:
+      - data:/root/.bitcoin
+      - data:/data
+    ports:
+      - "4000:4000"                                        # mainnet
+      - "14000:14000"                                      # testnet
+    command:
+      - start
+      - "--${COUNTERPARTY_NETWORK:-main}net"               # export COUNTERPARTY_NETWORK=test for testnet
```

Comment:
Hm, I still think this is going to be confusing... `NETWORK` means to me "mainnet" or "testnet", not "main" or "test".

Any ideas how to work around that?

Reply:
no sorry, I didn't find anything!